internal/driver/driver.go

```diff
@@ -184,23 +199,28 @@ outloop:
 			}
 
 		}
+		//4. Record all errors
```

Comment:
use `// 4. Reco...`

Reply:
I'm not sure if you mean to change the comments to this.
`//4. use s7_errors to record the abnormal error messages of all read points`